saleor/discount/migrations/0043_sale_notification_sent_datetime.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.13 on 2022-07-08 06:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("discount", "0042_migrate_orderdiscount_id_to_uuid"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="sale",
+            name="notification_sent_datetime",
+            field=models.DateTimeField(null=True, blank=True),
```

Comment:
Setting this to null here will fire up end hooks for all existing and ended Sales, am I right?
Assuming that I have 100 old sales, then this will trigger 100 webhooks that sale has ended, right?
Is it expected? Shouldn't we ignore the old sales here?

Reply:
Good point. So maybe as default we should set the current time?

Reply:
We can change to the current time with some explaining comments. I find this a bit confusing since `null` seems more appropriate value if there was no notification. But even if we set null and then Celery would trigger the tasks right after the deployment, we will end up having a similar date set in the database, but in the second case we actually fire events that no one listens to.